### PR TITLE
doc: add minutes for meeting 12 Nov 2020

### PR DESCRIPTION
**Pull request by mhdawson**
Signed-off-by: Michael Dawson <mdawson@devrus.com>

### CODE REVIEW

**Comment by mhdawson** [comment]
@nodejs/next-10 this could use at least 1 review so I can land.

File: meetings/2020-11-12.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# Node.js  Next 10 Years team Meeting 2020-11-12
```
Comment (trivikr) [inline meetings/2020-11-12.md:1]:
```suggestion
# Node.js Next 10 Years team Meeting 2020-11-12
```

File: meetings/2020-11-12.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+* **Recording**: https://youtu.be/3M6qN-u6CTM
+* **GitHub Issue**: https://github.com/nodejs/next-10/issues/32
+
+## Present
+
+* Michael Dawson (@mhdawson)
+* Geoffrey Booth (@GeoffreyBooth)
+* Wes Todd (@wesleytodd)
+* Mary Marchini
```
Comment (trivikr) [inline meetings/2020-11-12.md:13]:
```suggestion
* Mary Marchini (@mmarchini)
```

File: meetings/2020-11-12.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+* Christopher Hiller
```
Comment (trivikr) [inline meetings/2020-11-12.md:14]:
```suggestion
* Christopher Hiller (@boneskull)
```

**Review by bmeck — APPROVED** [review]

**Review by trivikr — APPROVED** [review]
LGTM, all are nits

**Pull request merged**